zerver/lib/cache.py

```diff
@@ -321,7 +321,7 @@ def user_profile_by_api_key_cache_key(api_key: str) -> str:
     'id', 'full_name', 'short_name', 'email',
     'avatar_source', 'avatar_version', 'is_active',
     'is_realm_admin', 'is_bot', 'realm_id', 'timezone',
-    'date_joined'
+    'date_joined', 'delivery_email'
 ]  # type: List[str]
```

Comment:
Can you explain why you made this change?  That is used for data sent down to all clients, primarily, and I don't think this field belongs here.